jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java

```diff
@@ -167,5 +190,7 @@ public void callback(final List<NodeInstanceSummary> details) {
         HTML getCorrelationKeyText();
 
         HTML getParentProcessInstanceIdText();
+        
+        HTML getSlaComplianceText();
```

Comment:
Returning the html element is a practice we're trying to get away, please update to only set the relevant text, something like _setSlaCompliance(String text)_